zabbix-mysql-dump

```diff
@@ -49,7 +54,7 @@
 # DEFAULT VALUES
 #
 # following will store the backup in a subdirectory of the current directory
-DUMPDIR="$(dirname "$(realpath -s "$0")")"
```

Comment:
It seems you are right. Perhaps DUMPDIR should be initiated with `.`? Can't see any reason for the current, seemingly over-engineered, situation.

I suggest you try your hand at a pull request for this problem. Hope your PR will be luckier than this one…  ;-]